mysql/datadog_checks/mysql/statement_samples.py

```diff
@@ -682,8 +715,13 @@ def _explain_statement(self, cursor, statement, schema, obfuscated_statement):
         except pymysql.err.DatabaseError as e:
             if len(e.args) != 2:
                 raise
+            db_explain_error = DBExplainError.invalid_schema
```

Comment:
`invalid_schema` is misleading here. The problem is that the agent didn't have access to the default schema on the connection that ran this query, not that the default schema was invalid. 

Maybe a better error name would be `use_schema_error`. Or `default_schema_access`?